infrastructure/ai/qdrant_vector_store.py

```
@@ -38,7 +38,7 @@ async def insert(
                 points=[point]
             )
         except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)